backend/social_tasks/tests/test_verifiers.py

```
@@ -0,0 +1,320 @@
+        )
+        self.task = SocialTask.objects.create(
+            slug='join-discord',
+            name='Join Discord',
+            category=self.category,
+            points=10,
+            verification_type='discord_guild_join',
+            action_url='https://discord.gg/x',
+            cta_text='Join',
+        )
+
+    def _link_discord(self, token_value='tok'):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_link_discord`

(ANN202)